pebbles/static/js/controllers/DashboardController.js

```diff
@@ -134,7 +134,10 @@ app.controller('DashboardController', ['$q', '$scope', '$routeParams', '$interva
                     }
                 }
                 $scope.instances = response;
-                DesktopNotifications.notifyInstanceLifetime(response);
+            });
+	    queryParams.show_only_mine = true;
```

Comment:
IMO, this is not the best efficient way as it queries the DB again and also creates readability problems when you change the queryParams in middle.
I would try to filter the instances based on user id in the loop which runs before this line using. Loadash library is already being used in the code, eg. var own_instances = _.filter($scope.instances, {'user_id': AuthService.getUserId()});